package.json

```diff
@@ -17,8 +17,10 @@
     "test": "jest",
     "lint": "prettier-check src/*.js",
     "format": "prettier --write src/*.js",
-    "watch": "watchify src/main.js --verbose --standalone envSpec -o dist/envspec.js",
+    "watch": "watchify src/main.js --ve.jsrbose --standalone envSpec -o dist/envspec.js",
```

Comment:
Oops! I see there is a typo here. Please change `--ve.jsrbose` back to `--verbose` please.